app/views/school-administrator/dashboard/SchoolAdminDashboardTeacherView.vue

```diff
@@ -47,6 +47,10 @@ export default {
         teacherId() {
           return this.$route.params.teacherId
         },
+        outcomesReportUrl () {
```

Comment:
shouldn't this be a method instead of computed?

Reply:
in these 2 vue it don't have input and always output a stable string. so i think computed is good to use